src/App.js

```diff
@@ -0,0 +1,11 @@
+import './App.css';
```

Comment:
![giphy-downsized](https://user-images.githubusercontent.com/93735137/155951395-e1ec66c7-3f4c-43b9-aae1-805b1f763bd2.gif)
Good Job so far.
Dependences are installed properly. Time for merge.